new-pipeline/src/dashboards.js

```diff
@@ -1081,6 +1081,23 @@ function getDescriptionLink(metric, channel, description) {
       class: "btn btn-outline-primary fa fa-info-circle",
     }).text(" More details"));
   }
-
   return link;
 }
+
+function getUseCounterLink(metric, channel, description) {
+  if (metric.startsWith("USE_COUNTER") !== true) {
+    return null;
+  }
+  else {
+    metricSplit = metric.split("_");
+    var group = metricSplit[2];
```

Comment:
What happens if the function gets e.g. passed a `metric` string "USE_COUNTER"?

Reply:
Hm. Could that happen? I suppose I can add a safeguard... something like:
```
if (metricSplit.length < 3) {
return null;
}
```